lerna.json

```diff
@@ -12,5 +12,6 @@
       ]
     }
   },
-  "useNx": true
+  "useNx": true,
+  "useWorkspaces": true
```

Comment:
This is a sneaky change.  With node 18 the lerna style workspaces was no longer working.  This changes the project to use npm workspaces.  This rolls all the common deps up into the the root node_modules directory.  I had to change where we find a couple of language serves but otherwise wasn't any impact.